packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -63,8 +65,16 @@ class BottomNavigationBarItem {
   final Widget activeIcon;
 
   /// The title of the item. If the title is not provided only the icon will be shown when not used in a Material Design [BottomNavigationBar].
+  @Deprecated('`title` was deprecated, use `label` instead.')
```

Comment:
This should follow the deprecation style: https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecation

Reply:
Also: Can you update the doc comment on this to mention that this property is deprecated and link to the one that should be used instead?

Reply:
Done.

Based on the breaking change guide you sent, I also created this breaking change proposal: flutter.dev/go/bottom-navigation-bar-title-deprecation, it has been open to discussion for a ~week or so now.